### PR TITLE
grpc: add an option to provide gRPC dial opts in Loopback func

### PR DESCRIPTION
**Pull request by edganiukov**
So it possible to configure custom credentials, etc.

### CODE REVIEW

**Comment by edganiukov** [comment]
hm, okay, that actually doesn't work. Looking how to fix.

**Comment by edganiukov** [comment]
For now changing only `Loopback` functions.

File: std/grpc/wire.go [hunk 1]
```diff
@@ -35,9 +35,17 @@ func ProvideConn(ctx context.Context, req *Backend) (*grpc.ClientConn, error) {
 	return client.Dial(ctx, endpoint, grpc.WithTransportCredentials(insecure.NewCredentials())) ///  XXX mTLS etc.
 }
 
-func Loopback(ctx context.Context) (*grpc.ClientConn, error) {
+func Loopback(ctx context.Context, grpcOpts ...grpc.DialOption) (*grpc.ClientConn, error) {
 	endpoint := fmt.Sprintf("127.0.0.1:%d", server.ListenPort())
```
Comment (attilaolah) [inline std/grpc/wire.go:39]: BTW why don't we use `net.JoinHostPort` for these?

**Review by attilaolah — APPROVED** [review]

File: std/grpc/wire.go (hunk 1 above)
Comment (edganiukov) [reply std/grpc/wire.go:39]: I guess here is not that bad as we explicitly use IPv4 (and  `server.ListenPort()` is an integer and would need to be converted to string to be use in `net.JoinHostPort`).

Comment (edganiukov) [reply std/grpc/wire.go:39]: and from what I heard people do not like `strconv.Itoa()`

Comment (edganiukov) [reply std/grpc/wire.go]: changed btw

Comment (attilaolah) [reply std/grpc/wire.go]: Yeah it sucks that the port must be a string in `net.JoinHostPort`, I forgot about that.

**Review by attilaolah — APPROVED** [review]

**Pull request merged**